app/presenters/work_presenter.rb

```diff
@@ -32,10 +48,6 @@ def keywords
     keywords_attributes.map(&:text).join(', ')
   end
 
-  def related_works
-    related_works_attributes.map(&:to_s)
-  end
-
```

Comment:
I was a bit surprised to see this since it's not obviously link-related. Am I wrong about that? Or is this handled elsewhere now?

Reply:
I'm surprised too. However, it isn't used anywhere, so maybe I was just removing cruft?